src/lint.js

```diff
@@ -27,7 +27,7 @@ module.exports = function (body, title, failureCallback, infoCallback) {
     } else {
         // Ensure a Jira ticket is referenced on the last line of the body
         infoCallback('Checking PR body Jira reference');
-        const jiraRegex = new RegExp('^(Relates to|Closes) ((?<!([A-Z]{1,10})-?)[A-Z]+-\\d+)$');
+        const jiraRegex = new RegExp('^(Relates to|Closes) (http|https):\/\/finder\.atlassian\.net\/browse\/[A-Za-z]+-[0-9]+$');
```

Comment:
Nice one

Reply:
Required refactor to support Markdown URL syntax.